Assets/Scripts/PlayerInteractionController.cs

```diff
@@ -157,7 +160,7 @@ private void SetThoughtForInteractableItem(InteractableItem item)
         }
         else
         {
-            thought.Add(new ItemBubble.StyledIcon(){Sprite = itemUseThought, Style = ItemBubble.Style.Solid});
+            thought.Add(new ItemBubble.StyledIcon(){Sprite = item.GetSprite(this)[0], Style = ItemBubble.Style.Solid});

```

Comment:
Result of `item.GetSprite(this)` may be directly passed to `thoughtBubble.RenderSprites(…)`. This seems highly unnecessary.